lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
@ayayalar could you make this 6 for now?  We'll change this over when we merge the PR for the new section.